2015/11/08 22:04:10 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not allowed to access this page.' in /var/www/yii/framework/gii/GiiModule.php:168
Stack trace:
#0 /var/www/yii/framework/web/CController.php(263): GiiModule->beforeControllerAction(Object(DefaultController), Object(CInlineAction))
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#3 /var/www/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/yii/SPVO/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/SPVO/index.php?r=gii
---
2015/11/08 22:25:42 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /var/www/yii/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/yii/SPVO/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/SPVO/index.php?r=gii/model
HTTP_REFERER=http://127.0.0.1/yii/SPVO/index.php?r=gii/controller
---
2015/11/08 22:25:47 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /var/www/yii/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/yii/SPVO/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/SPVO/index.php?r=gii/model
HTTP_REFERER=http://127.0.0.1/yii/SPVO/index.php?r=gii/controller
---
2015/11/08 22:26:12 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /var/www/yii/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/yii/SPVO/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/SPVO/index.php?r=gii/model
HTTP_REFERER=http://127.0.0.1/yii/SPVO/index.php?r=gii/controller
---
2015/11/08 22:30:35 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'SPVO.user' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `user`.
in /var/www/yii/SPVO/index.php (13)
